pkg/ir/spec.go

```diff
@@ -176,34 +172,6 @@ func (e *ExternalDetails) Address() *netset.IPBlock {
 	return e.ExternalAddrs
 }
 
-func (s *SubnetDetails) getOverlappingVPCs() []ID {
-	return []ID{s.VPC}
-}
-
-func (n *NifDetails) getOverlappingVPCs() []ID {
-	return []ID{n.VPC}
-}
-
-func (i *InstanceDetails) getOverlappingVPCs() []ID {
-	return []ID{i.VPC}
-}
```

Comment:
Seems like some resources (NIF, VSI, VPEReservedIPsDetails) no longer need to store their VPC.

Reply:
I think it might help solving #129